### PR TITLE
Symfony Process version bump

### PR DESCRIPTION
**Pull request by mgenereu**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,42 +6,26 @@ cache:
   directories:
     - $HOME/.composer/cache/files
 
-php: [5.3, 5.4, 5.5, 5.6, 7.0, 7.1, hhvm]
+php: [7.1]
```
Comment (stof) [inline .travis.yml]: dropping CI for all older releases is a no-go, especially when composer.json does not drop support for them

File: .travis.yml [hunk 2]
```diff
@@ -6,42 +6,26 @@ cache:
...
     - MINK_HOST=127.0.0.1:8002
 
-matrix:
-  include:
-    - php: 5.6
-      env: ZOMBIE_VERSION='@^2.0'
-    - php: 5.6
-      env:
-        - NODE_VERSION='5.4.1'
-        - ZOMBIE_VERSION='@^4.0'
```
Comment (stof) [inline .travis.yml]: why dropping CI jobs for older versions ?

**Comment by stof** [comment]
Please revert all changes done in the CI config basically.

**Comment by mgenereu** [comment]
@stof Didn't mean to bug you.  Symfony Process is PHP 7.1 and up only and I was just trying to get the tests passing again. I'm behind a firewall and was trying to get Travis CI to work quickly.  I'll blow this all out in a few minutes once you get this message.

**Pull request closed without merging**